OmiseSwift/API Models/Source/SourceType.swift

```diff
@@ -92,6 +93,132 @@ public enum MobileBanking: RawRepresentable, Equatable, Hashable {
     case unknown(String)
 }
 
+public struct FPX: Codable, Equatable , Hashable{
+    public let email: String?
+    public let bank: FPXBank
+    
+    public init(bank: FPXBank,email: String? = nil){
```

Comment:
Is it very unlikely that FPX has an email? Otherwise I would not make the default value `nil`. But when most cases are without an email it's okay like this.

Reply:
I think most case it will have no email, on omise.js the email field is optional so most customer will skip it. In this case I think it should be the same.

Reply:
Okay than it's fine like this.